src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -28,10 +38,32 @@ public void findAllBusinesses() {
     businesses.add(new Listing("The Shiny Bean Coffee & Tea", new MapLocation(40.496328, -79.944862), 4.9, "https://goo.gl/maps/AYH2QCL7pkoMBxHA8", "https://theshinybean.com/"));
     businesses.add(new Listing("Weisshouse", new MapLocation(40.456684, -79.925499), 4.3, "https://goo.gl/maps/7tuXn7QF2hh7ioGYA", "https://www.weisshouse.com/"));
```

Comment:
I realize this is the result of a previous pull request, but please move these listings to a final ArrayList at the top of this object. That will allow using loop constructs here now, and moving forward.

Reply:
This is from Lindsey's code in the master branch, that is why I left it here for now

Reply:
Being that this is from a different push, feel free to create an issue to track it, and reference it here as a comment.

Reply:
#19 This hard coded businesses will be replaced with Lindsey's code of using the Places API to return the businesses in the surrounding area.

Reply:
Issue #19  This will removed by Lindsey's code